dataflow/src/main/java/org/checkerframework/dataflow/livevariable/LiveVar.java

```diff
@@ -0,0 +1,50 @@
+package org.checkerframework.dataflow.livevariable;
+
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.checkerframework.dataflow.analysis.AbstractValue;
+import org.checkerframework.dataflow.cfg.node.Node;
+import org.checkerframework.javacutil.BugInCF;
+
+/** A live variable wrapper turning node into abstract value. */
```

Comment:
A "live variable wrapper" would wrap a live variable, but this is wrapping a node.  Could you please improve this documentation?